src/containers/layouts/dashboards/trainings/trainings1/easy1.component.tsx

```diff
@@ -0,0 +1,165 @@
+import React from 'react';
+import {
+  ListRenderItemInfo,
+  ScrollView,
+  View,
+  ViewProps,
+} from 'react-native';
+import {
+  withStyles,
+  ThemeType,
+  ThemedComponentProps,
+  StyleType,
+} from '@kitten/theme';
+import { Exercise } from '@src/core/model/exercise.model';
+import {
+  TrainingCard1,
+  TrainingCardProps,
+} from '@src/components/trainings/trainingCard.component';
+import {
+  ViewPager,
+  List,
+} from '@kitten/ui';
+import { Text } from '@src/components/common';
+
+interface ComponentProps {
+  exercises: Exercise[];
+  onTrainingDetails: (index: number) => void;
+}
+
+interface State {
+  selectedPagerIndex: number;
+}
+
+export type Easy1ComponentProps = ThemedComponentProps & ComponentProps;
+
+class Easy1Component extends React.Component<Easy1ComponentProps, State> {
+
+  public state: State = {
+    selectedPagerIndex: 0,
+  };
+
+  private onChangePager = (index: number): void => {
+    this.setState({ selectedPagerIndex: index });
+  };
+
+  private onTrainingDetails = (index: number): void => {
+    this.props.onTrainingDetails(index);
+  };
+
+  private renderPagerIndicator = (index: number): React.ReactElement<ViewProps> => {
+    const { themedStyle, exercises } = this.props;
+    const additionalStyle: StyleType = index === this.state.selectedPagerIndex ?
+      themedStyle.pagerIndicatorSelected : null;
+    const marginStyle: StyleType = index === exercises.length - 1 ?
+      null : themedStyle.indicatorMarginRight;
+
+    return (
+      <View
+        style={[themedStyle.pagerIndicator, additionalStyle, marginStyle]}
+        key={index}
+      />
+    );
+  };
+
+  private renderPagerCard = (exercise: Exercise, i: number): React.ReactElement<TrainingCardProps> => {
+    const { themedStyle, exercises } = this.props;
+    const marginStyle: StyleType = i === exercises.length - 1 ?
```

Comment:
such cases can be achieved by passing bottom or vertical padding into container element

Reply:
Read the code :) this is horizontal paging.